### DISCUSSION TITLE
HELLOOOO.....

### DISCUSSION THREAD

**Discussion by SACH338** [Cyfrin/foundry-full-course-cu#2147] [General]
..

**Comment by EngrPips** [comment]
Helloooooooooo...............

**Comment by SACH338** [comment]
Recently Indian crypto exchanges wazirx said approximately 45%(as per preliminary working) of its crypto assets was affected by a cyber attack on July 18 ,that led to loss more than $230 million from one of its wallets 

Wazirx boasts a strong and robust block chain infrastructure 

Why this wallet is hacked ,it's also block chain infrastructure 
??


In a basic of block chain I learnt block chain systems never hacked 

But why it's hacked

**Reply by EngrPips** [reply]
Hello @SACH338, I think you should do more findings on what `Wazirx` truly is and the hack, the vulnerability that led to the hack and you might be more clear on why it gets hacked. Whenever there is a bug then there is room for a hack.

**Reply by SACH338** [reply]
👍
